src/WPF/ArcGISRuntime.WPF.Samples/Samples/Data/GenerateGeodatabase/README.md

```diff
@@ -2,11 +2,12 @@
 
 This sample demonstrates how to take a feature service offline by generating a geodatabase.
 
-
 <img src="GenerateGeodatabase.jpg" width="350"/>
 
 ### Instructions
 
 1. Pan and zoom to the area you would like to download features for, ensuring that all features are within the rectangle.
 2. Tap on the button. This will start the process of generating the offline geodatabase.
 3. Observe that the sample unregisters the geodatabase. This is best practice when changes won't be edited and synced back to the service.
+
+Note that the basemap will be automatically downloaded from an ArcGIS Online portal.
```

Comment:
Yes, because there isn't a straightforward way for an individual sample to be notified that the offline data was downloaded. The sample needs to check if the file is already present and request that the offline data is downloaded otherwise. On WPF the user is still presented with the option to download the data first, but the sample is coded to avoid a crash if the user doesn't take action.